bokeh/models/widgets/markups.py

```diff
@@ -76,6 +80,11 @@ class Markup(Widget):
     Raw CSS style declaration. Note this may be web browser dependent.
     """)
 
+    disable_math = Bool(False, help="""
```

Comment:
If I understand this correctly, this default will make bokeh always include mathjax if any markup is present? If that's the case, it's the other way that it should be.

Reply:
You say it should be a default not processed for TeX input?

We discussed this UX before 
https://github.com/bokeh/bokeh/pull/11585#issuecomment-917338525

Reply:
> If I understand this correctly, this default will make bokeh always include mathjax if any markup is present

yes, the expectation from users will be that it "just works" so this is the correct default behavior.

Reply:
It makes sense when coupled with `if contains_tex_string(model.text):`, which I didn't realise is the case. In this case it's fine. The point is that mathjax bundle shouldn't be loaded unnecessarily (given its size; by that I no only mean the download size, but also, and perhaps even more importantly, JS parsing/load time, which is significant).

Reply:
Perhaps this property could be called `verbatim`, as in do not apply any additional processing to the contents of the `div`.

Reply:
I'm fine with a different name tho I wonder if users might think `verbatim` relates to `pre`

Reply:
I think `verbatim` would be a good replacement for `render_as_text`, since `disable_math` can still have `html` markups processed.